recipes-bsp/u-boot/u-boot-imx_2018.03.bb

```diff
@@ -1,6 +1,6 @@
 # Copyright (C) 2013-2016 Freescale Semiconductor
-# Copyright 2017-2018 NXP
 # Copyright 2018 (C) O.S. Systems Software LTDA.
+# Copyright (C) 2017-2019 NXP
 
```

Comment:
Only because you are changing the Copyright. I would suggest you to use:

#Copyright 2017-2019 NXP

only. No need to add (C). If this is the only change needed in this PR I would accept it as is though.

Reply:
Since I am updating the years in the copyright line, I wanted to also update the style and add (c), which is more MIT and GPL like and was previously used with Freescale. Now we have some files with 3 different styles.
It was just a thought, but ok I can drop this.

Reply:
It's not a big deal anyway